spec/requests/api/v1/split_details_spec.rb

```diff
@@ -21,7 +21,8 @@
         split: split_with_details,
         variant: 'disabled',
         display_name: 'fantastic_split_with_information is off',
-        description: 'This feature makes nothing happen.'
+        description: 'This feature makes nothing happen.',
+        screenshot_file_name: 'screenshot.jpg'
```

Comment:
This seems like an awkward way to stub in an attachment. I think you should probably be using the screenshot setter with an actual fixture file if you want to test that the paperclip integration works as expected?